UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformationAdmin.Web/Controllers/HomeController.cs

```diff
@@ -24,5 +24,12 @@ public IActionResult Error()
             _logger.LogError(EventIds.SystemError.ToEventId(), "System error has occurred while processing request with exception:{ex}, at exception path:{path} for correlationId:{correlationId}", exceptionDetails?.Error.Message, exceptionDetails?.Path, correlationId);
             return View();
         }
+
+        [Route("/accessdenied")]
+        public IActionResult AccessDenied()
+        {
+            _logger.LogError(EventIds.UnauthorizedAccess.ToEventId(), "Unauthorized page requested by user: {user}", User.Identity.Name);
```

Comment:
Please include correlationId in logs and also pass it to view as its used in email subject.

Reply:
As confirmed in show & tell, no need to add CorrelationId.